src/languages/en.ts

```diff
@@ -2763,6 +2763,16 @@ export default {
         },
         copyReferralLink: 'Copy invite link',
     },
+    onboardingBottomMessage: {
```

Comment:
I propose to name this `systemChatFooterMessage`

Reply:
Done - 62b193a12ef1e23b8a31c8fe604100d622777565.